alena_automation/XPathExercises - Alena2.csv

```diff
@@ -0,0 +1,34 @@
+Xpath Tasks,Authors.xml,Authors2.xml (with namespaces)
+Normal:,,
+all elements,/authors,
+"all ""author"" elements",//author,
+"the root element ""authors"" - 2 variants","/* ,  authors, child::authors",
+"the same as 3,  but this time without using the name of the element",/*,
+all elements enclosed by elements  /authors/author,/authors/author,
+"all ""name"" elements (without using names of parent elements) ",//name,
+all elements which have 1 ancestors,,
+"all ""period"" attributes of ""author"" elements - 2 variants","//author[@period] , ",
+"all ""author"" elements with a ""period"" attribute. ",//author[@period],
+"the fourth ""author"" element. ",//author[4],
+"the ""name"" node of the fourth author",//author[4]/name,
+the name of the fourth author (text),//author[4]/name[text()],
+all elements that have any attributes,//*[@*],
+all elements without any attributes,/*/*/*[not(@*)],
+the node of author born in 1828. ,//author[@born='1828'],
+name of the author born in 1828 (text),//author[@born='1828']/name,
+all authors that died after the year 0. ,//author[@died>'0'],
+the second author that died after the year 0 ,//author[@died<'0']/name  --- not Done,
+the name of second author that died after the year 0 (text),,
+"the last ""author"" element. ",//author[last()],
+"the ""name"" element that contains the text ""Tolstoy"". ","//*[contains(name,'Tolstoy')]  ",
+"all elements that contain the text ""Tolstoy""",,
+all descendants of all authors that died before the year 0. ,,
+all elements name of which starts with letter a,"//*[starts-with(name,'A')]",
+"all elements name of which contains ""t""","//*[contains(name,'t')]  ",
+select name and nationality of the second author (text),//author[name='Leo Tolstoy']/name | //author[name='Leo Tolstoy']/nationality,
+Advanced:,,
+ Tolstoy's nationality. ,//name[@idd='LT'],
```

Comment:
it returns name nod, but needs to return nationality

Reply:
//author[name='Leo Tolstoy']/nationality